pandas/_libs/tslibs/offsets.pyx

```diff
@@ -4691,7 +4669,27 @@ def _get_offset(name: str) -> BaseOffset:
     --------
     _get_offset('EOM') --> BMonthEnd(1)
     """
-    if name.lower() not in _dont_uppercase:
+    if (
+        name not in _lite_rule_alias
+        and (name.upper() in _lite_rule_alias)
+        and name != "ms"
```

Comment:
separate to this PR, but can we move `'ms'`, `'us'`, `'ns'`, and `'min'` from `_lite_rule_alias`? `_lit_rule_alias` is only ever used with `.get(name, name)`, so I don't think they need to be in it?

Reply:
Thank you, I did as you suggested and removed `"min", ”ms”, “ns", and “us"` from `_lite_rule_alias` in separate PR #56516. I added these aliases to the list _dont_uppercase, because  otherwise we uppercase them in  `_get_offset()`. could you please take a look at this PR?

I am not sure, do we need “`Min”` in `_lite_rule_alias`?  We want to deprecate the alias “Min” and left only lowercase “min” for Minutes. Am I correct?